test/TestStablePriceOracle.js

```diff
@@ -1,12 +1,21 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
 const DummyOracle = artifacts.require('./DummyOracle');
 const StablePriceOracle = artifacts.require('./StablePriceOracle');
 
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
 const toBN = require('web3-utils').toBN;
 
 contract('StablePriceOracle', function (accounts) {
     let priceOracle;
 
     before(async () => {
+		ens = await ENS.new();
+		registrar = await BaseRegistrar.new(ens.address, namehash.hash('eth'));
+		await ens.setSubnodeOwner('0x0', sha3('eth'), registrar.address);
+		await registrar.addController(accounts[0]);
```

Comment:
Please fix indentation

Reply:
Why did you add `ens` and `registrar` when neither are used in the test (Commenting out all the lines you added still pass the test). If you do actually need them, please add `const ens` and `const registrar`